MusicNotationKit/MusicNotationKit/Measure.swift

```diff
@@ -46,12 +46,36 @@ public struct Measure: ImmutableMeasure, Equatable {
         noteCount = immutableMeasure.noteCount
         recomputeNoteCollectionIndexes()
     }
-
+	
+	public func note(at index: Int, inSet setIndex: Int) throws -> Note {
```

Comment:
For now, I am making a default value of 0 for every `setIndex` so let's stick with that.